torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -585,38 +585,49 @@ struct GraphExecutorImpl : public GraphExecutorImplBase {
     // Phase 0. Inline functions, then clean up any artifacts that the inliner
     //          left in that may inhibit optimization
     Inline(*opt_graph);
-    GRAPH_DUMP("After Inline, before LowerGradOf", opt_graph);
+    GRAPH_DEBUG("After Inline, before LowerGradOf\n", opt_graph->toString());
     LowerGradOf(*opt_graph);
-    GRAPH_DUMP("After LowerGradOf, before specializeAutogradZero", opt_graph);
+    GRAPH_DEBUG(
+        "After LowerGradOf, before specializeAutogradZero\n",
+        opt_graph->toString());
```

Comment:
Does it work to use `*opt_graph` here instead of `opt_graph->toString()`?  It's maybe slightly cleaner that way.

Reply:
I agree, that does look much cleaner and it also works! I'll make the changes